mungegithub/submit-queue/deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         - /mungegithub
         - --dry-run=true
         - --stderrthreshold=INFO
+        - --period=20s
```

Comment:
Well, the "since" code is hard-coded, so they will all have to use it anyway. Maybe I should add the key to test-infra? Or maybe I should make it possible to use since or not. OK I'll do that ...